runtime-modules/storage/src/data_directory.rs

```diff
@@ -40,6 +40,12 @@ use crate::data_object_type_registry;
 use crate::data_object_type_registry::IsActiveDataObjectType;
 use crate::{MemberId, StorageProviderId, StorageWorkingGroup, StorageWorkingGroupInstance};
 
+// Temporary representation.
+type ChannelId = u64;
```

Comment:
Both of these types need to be configured, because they have to match types used in other modules.

Reply:
Agree, they should be generic in data structures. Though we don`t have DAOs and new content directory pallets yet to be able to refer them.

Reply:
https://github.com/Joystream/joystream/pull/2099/commits/6f7481efd0170f63797ed930f5bf74e251773ac5

Reply:
I think you misunderstood what I meant here, I meant the type should come from the runtime via the `Trait`.

Reply:
Nevertheless, i assume that they would come via trait inheritance mechanism from other runtime pallet `Trait`s.
Similar to 

```
// Alias for the member id.
pub(crate) type MemberId<T> = <T as membership::Trait>::MemberId;

/// Storage provider is a worker from the working group module.
pub type StorageProviderId<T> = working_group::WorkerId<T>;
```

```
pub trait Trait:
    pallet_timestamp::Trait
 ....
    + membership::Trait
    + working_group::Trait<StorageWorkingGroupInstance>
{
```

Reply:
Does it make sense?

Reply:
ok.